dashboard/models/user.py

```diff
@@ -89,10 +88,20 @@ def user_detail(self, email, field='photo'):
 class User(object):
     def __init__(self, session, app_config):
         """Constructor takes user session."""
-        self.userinfo = session.get('userinfo', None)
+        self.id_token = session.get('id_token', None)
         self.app_config = app_config
         m = Mozillians(self.app_config)
-        self.profile = m.user_detail(self.userinfo.get('email'))
+        self.userinfo = session.get('userinfo')
+        self.idvault_info = session.get('idvault_userinfo')
+        self.profile = m.user_detail(self.email())
+
+    def email(self):
+        try:
+            email = self.userinfo.get('email')
+        except Exception as e:
```

Comment:
this currently wont work in dev (and in prod on thursday), that is, it will find `userinfo.email` just fine and work, but if ever it would fall back due to the exception, it will fail to load `https://sso.mozilla.com/claim/emails` because it is no longer sent as part of reducing the id_token
That value is available from PersonAPI though